docs/install.rst

```diff
@@ -164,6 +166,12 @@ There may be a delay of a day or two between when a new version of ``astropy``
 is released and when a package is available for Anaconda. You can check
 for the list of available versions with ``conda search astropy``.
 
+To install all of the optional dependencies using ``conda`` enter the following::
+
+    conda install -c pkgs/main -c conda-forge \
+      scipy h5py beautifulsoup4 html5lib bleach pyyaml pandas sortedcontainers \
+      pytz matplotlib setuptools mpmath bottleneck jplephem asdf bintrees
```

Comment:
Why still install bintrees when we warn that it is no longer maintained above?

Reply:
Yeah, I thought about that. It is still supported in the code and listed as an optional dependency above, so it seems like it belongs in this list.  In any case this is just a small package with no other dependencies.

Reply:
I think we have an unspoken policy to not actively advertise deprecated things. I would prefer if it is removed, as people usually blindly copy-paste, but if you insist it should stay on the command, then so be it.

Reply:
I'll change, I was on the fence.